content/en/docs/contribute/start.md

```diff
@@ -210,15 +210,11 @@ to base your work on. Use these guidelines to make the decision:
     periodically merge these to `master`. This kind of branch has a name like
     dev-\<version>-\<language code>.\<team milestone>; for example:
     `dev-{{< release-branch >}}-ja.1`.
-- If you're writing or updating documentation for a feature change release,
```

Comment:
@jimangel , nit (not part of your changes, but I read the content),
Line 197 seems confusing. Can you use other branches besides `master` to make changes to published content?
Can you submit changes to a prior published release branch (snapshot)?

Reply:
No, it should be `master` for everything unless it's an upcoming feature, then it's `dev-x.xx`. However there are a _few_ exceptions:

- Localization teams working on a release that is not the current one can commit to `release-x.xx` (where `release-x.xx` does not equal `master`).
- Very rarely, a CVE / security / functional issue will arise where it make sense to PR across all supported `release-x.xx` branches.

I agree, there can be some improvements here; maybe in another PR. Hope that helps!